tests/unit/browser/webkit/test_webkitelem.py

```diff
@@ -130,6 +130,7 @@ def _style_property(name, strategy):
     elem.styleProperty.side_effect = _style_property
     wrapped = webkitelem.WebKitElement(elem, tab=None)
     return wrapped
+    # pylint: enable=too-many-locals,too-many-branches
```

Comment:
No need for this, as pylint already only turns things off for this function and it's needed for the entire function.